CruzHacks/cruzhacks-2021-website
# footer component

Problem
=======
As a prospective hacker,
I would like to view additional footer resources
To resolve more inquiries and questions I have about CruzHacks.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199382769817337/f)

Solution
========
To implement the Footer section of the homepage, flexbox-based positioning and margin alignments were used to replicate the provided Figma design with responsiveness. Although preliminary social media links and other graphics have not been included yet, this first draft outlines boilerplate code regarding how the footer is to be implemented.

Change summary:
---------------
* first iteration of footer https://github.com/CruzHacks/cruzhacks-2021-website/commit/f8fd13933a8fe9f65957f78fbe512cc41ffbf231

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/footer`
2. Start local development server using `yarn start`
3. Verify that footer section is designed and responds correctly.
